Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls.WPF/InkToolbar/InkToolbarToolButton.cs

```diff
@@ -3,41 +3,30 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
-using System.Collections.Generic;
-using System.Collections.ObjectModel;
-using System.ComponentModel;
-using System.Linq;
 using System.Windows;
-using System.Windows.Controls;
-using System.Windows.Markup;
-using Microsoft.Toolkit.Win32.UI.Interop;
-using Windows.Foundation;
-using Windows.UI.Xaml.Media;
-using uwpControls = global::Windows.UI.Xaml.Controls;
-using uwpInking = Windows.UI.Input.Inking;
-using uwpXaml = global::Windows.UI.Xaml;
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT;
 
 namespace Microsoft.Toolkit.Win32.UI.Controls.WPF
 {
     /// <summary>
-    /// Wpf-enabled wrapper for <see cref="global::Windows.UI.Xaml.Controls.InkToolbarToolButton"/>
+    /// Wpf-enabled wrapper for <see cref="Windows.UI.Xaml.Controls.InkToolbarToolButton"/>
     /// </summary>
     public class InkToolbarToolButton : WindowsXamlHostBaseExt
     {
-        internal global::Windows.UI.Xaml.Controls.InkToolbarToolButton UwpControl => this.XamlRootInternal as global::Windows.UI.Xaml.Controls.InkToolbarToolButton;
```

Comment:
In most cases cleaning up the using statements removed the need to disambiguate with `global::` namespace. It should be rare that such an alias is required to be used.